nixpkgs/nixpkgs.bzl

```diff
@@ -890,6 +890,18 @@ def nixpkgs_cc_configure_deprecated(
 
 def _nixpkgs_python_toolchain_impl(repository_ctx):
     cpu = get_cpu_value(repository_ctx)
+    os = {"darwin": "osx"}.get(cpu, "linux")
+
+    if repository_ctx.attr.target_constraints == None and repository_ctx.attr.exec_constraints == None:
```

Comment:
You can use `if not variable and not other_variable` to cover uscases of empty strings and null-like values.

Reply:
empty list is not valid for specification by the consumer?

Reply:
This is just a soft-ball comment, Starlark is Pythonesque and  I think it is generally a bad smell to use `== None` in Python .